Api version + renamed events (#49)
flant/shell-operator: HOOKS.md

- `name` is an optional identifier. It is used to distinguish different bindings during runtime. For more info see [binding context](#binding-context).

> - `apiVersion` is an optional group and version of object API.

Reviewer: 👍

There are some extra fields for `onKubernetesEvent`-type events:

- `watchEvent` — the event type is identical to the values in the `event` parameter: “Added”, “Modified” or “Deleted”.
- `object` — an object related to event.

> - `filterResult` — result of jq execution with specified `jqFilter` expression. If `jqFilter` is not specified, then filterResult is omitted.

Reviewer: ```suggestion
- `filterResult` — the result of jq execution with specified `jqFilter` on the abovementioned object. If `jqFilter` is not specified, then `filterResult` is omitted.
```

#### schedule binding context example

> Let's hook have this schedule configuration:

Reviewer: ```suggestion
For example, if you have the following binding configuration of a hook:
```
